skynet/modules/ttt/utils.py

```diff
@@ -0,0 +1,25 @@
+from skynet.modules.ttt.assistant.constants import assistant_default_system_message, assistant_limit_data_to_rag
```

Comment:
Maybe call this file rag_utils?

Reply:
oh, i mistakenly put it directly in ttt/, wanted to place it in ttt/assistant